en/custom_build/mavlink.md

```diff
@@ -1,12 +1,17 @@
 # MAVLink Customisation
 
 QGC communicates with flight stacks using [MAVLink](https://mavlink.io/en/), a very lightweight messaging protocol that has been designed for the drone ecosystem.
-QGC includes the [ArduPilotMega.xml](https://mavlink.io/en/messages/ardupilotmega.html) dialect by default, which allows it to communicate with both PX4 and Ardupilot (PX4 uses [common.xml](https://mavlink.io/en/messages/common.html), which is incuded in ArduPilotMega).
 
-In order to add support for a new set of messages you will ultimately need to add them to `ArduPilotMega.xml` or `common.xml`, or fork *QGroundControl* and include your own dialect.
+QGC includes the [all.xml](https://mavlink.io/en/messages/all.html) dialect by default from QGC 4.2.8, which includes all other dialects in the [mavlink/mavlink](https://github.com/mavlink/mavlink/tree/master/message_definitions/v1.0 ) repository, and allows it to communicate with both PX4 and Ardupilot.
```

Comment:
@DonLakeFlyer Is this the right version of QGC? Otherwise is this OK?

I assume that ArduPilot might use development.xml too, but don't state that.

Reply:
I'm assuming these two commit represent the change that @DonLakeFlyer mention: 
- https://github.com/mavlink/qgroundcontrol/commit/c716fbb28383df9984f9efd84c70e6fb39390d28
- https://github.com/mavlink/qgroundcontrol/commit/cb39702cc6ba729333d139c1bd6047255de11d76
I think it was accidental that the v4.2.8 worked with the `all.xml` since it include the `ArduPilot.xml`.

So I would say something like this:
```suggestion
QGC includes the [all.xml](https://mavlink.io/en/messages/all.html) dialect by default. The `all.xml` includes all other dialects in the [mavlink/mavlink](https://github.com/mavlink/mavlink/tree/master/message_definitions/v1.0 ) repository, and allows it to communicate with both PX4 and Ardupilot. Previous versions of QGC, since v4.2.8, used the `ArduPilotMega.xml` dialect.
```


That way, we explained that the previous version used the `ArduPilotMega.xml` and that the future release will use the `all.xml`. Feel free to rephrase it.